src/utils/term.ts

```diff
@@ -0,0 +1,14 @@
+export class Term {
+
+    public term : string;
```

Comment:
This should ideally be an `Array` as we're going to have to deal with synonymes. I would also rename the class to `Entity`. The ids map to entities if I understand correctly and not to a singe term, but to a list of synonymes. We might have to check that with @MusicConnectionMachine/group-1 though.